Lacework/manifest.json

```
@@ -3,32 +3,32 @@
            "title": "Lacework",
            "type": "object",
            "properties": {
-               "access_key": {
-                   "description": "Client Identifier",
+               "key_id": {
+                   "description": "The KeyId of your API Key",
                    "type": "string"
                },
-               "secret_key": {
-                   "description": "Client Secret",
+               "secret": {
+                   "description": "The secret of your API Key",
                    "type": "string"
                },
-               "lacework_url": {
-                   "description": "Client Lacework application name",
+               "account": {
+                   "description": "The account of your API Key",
```

```suggestion
                   "description": "The account of your API Key (e.g: `YourLaceworkTenant.lacework.net`)",
```
Perhaps, we should provide example here as well?